src/help_window.rs

```diff
@@ -48,16 +51,15 @@ impl HelpWindow {
             "Run 'donkeytype help' for more options",
         ];
 
-        // Render Clear widget first
-        frame.render_widget(Clear, area);
-
         // Render block
         frame.render_widget(block, area);
 
         // Render text paragraphs
         for (i, &text) in help_text.iter().enumerate() {
             let paragraph = Paragraph::new(text).style(Style::default().fg(Color::White));
-            frame.render_widget(paragraph, chunks[i]);
+            if i < chunks.len() {
+                frame.render_widget(paragraph, chunks[i]);
+            }
```

Comment:
maybe consider simply creating the right amount of chunks, the same as the number of elements in help_text vector?

Reply:
done.